feature_server/commands.py

```diff
@@ -524,6 +560,11 @@ def god(connection, value = None):
         connection.god_build = connection.god
     else:
         connection.god_build = False
+    return 'You have entered god mode.'
+
+@admin
+def god(connection, value = None):
+    godsilent(connection, value)
```

Comment:
It's done the other way around for e.g. `/gotos`, so this might want to be adjusted.

On the other hand, the question is if this command should really exist, since I presume the reason the chat message was there originally was to prevent admin cheating.